Dockerfile

```diff
@@ -234,7 +235,21 @@ CMD ["semgrep", "--help"]
 LABEL maintainer="support@semgrep.com"
 
 ###############################################################################
-# Step 4 (optional) nonroot variant
+# Step4: install semgrep-pro
+###############################################################################
+# This step is valid only when run from Github Actions.
+# See .github/workflows/build-test-docker.yaml
+
+#coupling: the 'semgrep-cli' name is used in release.jsonnet
+FROM semgrep-oss AS semgrep-cli
+
+# A semgrep docker image with semgrep-pro already included in the image,
+# to save time in CI as one does not need to wait 2min each time to
+# download it.
+RUN --mount=type=secret,id=SEMGREP_APP_TOKEN SEMGREP_APP_TOKEN=$(cat /run/secrets/SEMGREP_APP_TOKEN) semgrep install-semgrep-pro
```

Comment:
@tpetr suggested something like `[[ -z "${SEMGREP_APP_TOKEN}" ]] || semgrep install-semgrep-pro` so that the image would build even if `{SEMGREP_APP_TOKEN` isn't available. IMO this is a minor issue, if need be I'd better ship this one fast.